assets/templates/partials/filter_old.tmpl

```diff
@@ -0,0 +1,52 @@
+{{ $filter := .Data.Filter }}
+{{ $response := .Data.Response }}
+{{ $lang := .Language }}
+<form id="form" class="js-auto-submit__form" action="">
+    <div class="col col--md-12 col--lg-18 margin-bottom">
+        <div class="margin-bottom-md--2 flush-col js-mobile-filters">
+            
```

Comment:
worth removing this empty line as you should not have a space between html elements